### DISCUSSION TITLE
NFT Minting - DevOpsTools - member "get_most_recent_deployment" not found or not visible.

### DISCUSSION THREAD

**Discussion by zrmonty** [Cyfrin/foundry-full-course-cu#1201] [General]
![Screenshot from 2023-12-13 11-33-27](https://github.com/Cyfrin/foundry-full-course-f23/assets/122672823/70b266a5-f692-418c-8c33-ea8edb9b8ac4)

I just ran into this issue on the NFT course during the interactions.s.sol setup. I've been kicking it around a bit trying to figure out what happened. Hopped into the actual contract and the name is correct as I copied the function name over. I double checked the import and am using the relative path. 

Not sure, what other stones I should flip over here.

**Comment by ghost** [comment]
By mistake you passed the contract instead of just the name.

Update to:

```javascript
address mostRecentDeployed = DevOpsTools.get_most_recent_deployment("BasicNft", block.chainid)
```

**Reply by Abbasjafri-syed** [reply]
thanks that resolved the issue...
